CodeStarterCamp_Week4/BodyCondition.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  BodyCondition.swift
+//  CodeStarterCamp_Week4
+//
+//  Created by Hyungmin Lee on 2023/03/14.
+//
+
+import Foundation
+
+class BodyCondition {
+    var upperBodyStrength: Int {
+        didSet (oldValue) {
+            print("상체근력이 \(upperBodyStrength - oldValue) 상승합니다.")
+        }
+    }
+    var lowerBodyStrength: Int {
+        didSet (oldValue) {
+            print("하체근력이 \(lowerBodyStrength - oldValue) 상승합니다.")
+        }
+    }
+    var muscleEndurance: Int {
+        didSet (oldValue) {
+            print("근지구력 \(muscleEndurance - oldValue) 상승합니다.")
+        }
+    }
+    var fatigue: Int {
+        didSet (oldValue) {
+            if fatigue > oldValue {
+                print("피로도가 \(fatigue - oldValue) 상승합니다.")
+            }
+            else {
+                print("피로도가 \(oldValue - fatigue) 하락합니다.")
+            }
+        }
+    }
```

Comment:
```suggestion
    var fatigue: Int {
        didSet (oldValue) {
            fatigue > oldValue ? print("피로도가 \(fatigue - oldValue) 상승합니다.") : print("피로도가 \(oldValue - fatigue) 하락합니다.")
        }
    }
```

이 부분은 순전히 취향 차이라는 생각이 드는데, 저는 코드 뎁스가 깊어지는 걸 지양하는 편이라 이럴 때는 `삼항연산자`를 활용합니다!

Reply:
이부분은 삼항연산자로 했을 때가 더 직관적일 수 있다는 생각이 들어요! 이유는 언급해주셨듯이 뎁스가 줄어서!